README.md

```diff
@@ -13,8 +13,17 @@ Newsboat [![Cirrus CI Build Status](https://api.cirrus-ci.com/github/newsboat/ne
 Newsboat is an RSS/Atom feed reader for the text console. It's an actively
 maintained fork of Newsbeuter.
 
-Logo [by noobilanderi](https://groups.google.com/forum/#!topic/newsboat/Xm5pTsbeMEk),
-licensed under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/).
+A feed reader pulls updates directly from sites like blogs and news agencies,
+and lets you review them in a single interface. Many times, the feed includes
+the full text of the update, so you don't even need to start a web browser! You
+can learn more about feed readers [in
+Wikipedia](https://en.wikipedia.org/wiki/News_aggregator).
```

Comment:
Nitpick: To me, `on Wikipedia` sounds more natural than `in Wikipedia`

Reply:
Right! Fixed by force-pushing. Thanks!